src/chain/network_event.rs

```diff
@@ -65,6 +65,8 @@ pub enum NetworkEvent {
 
     /// A list of proofs for a neighbour section, starting from the current section.
     ProvingSections(Vec<ProvingSection>, SectionInfo),
+
+    UpdateSharedState(SectionInfo),
```

Comment:
Maybe can be more specific with name. could be AckMessage (similar to how we do for ExpectCandidate).